client/src/main/java/io/split/service/SplitHttpClientImpl.java

```diff
@@ -33,7 +33,7 @@ public static SplitHttpClientImpl create(
         return new SplitHttpClientImpl(client, requestDecorator);
     }
 
-    private SplitHttpClientImpl
+    public SplitHttpClientImpl
```

Comment:
Why did you make this method public? You can use the create method.

Reply:
Some tests create the httpclient outside methods, which require try catch block, so I made it public, I will updated these tests and revert back to private.